src/Utilities/universalfallback.jl

```diff
@@ -268,6 +281,52 @@ function MOI.get(uf::UniversalFallback, listattr::MOI.ListOfConstraintAttributes
     return list
 end
 
+# Objective
+MOI.supports(uf::UniversalFallback, ::MOI.ObjectiveSense) = true
+function MOI.set(uf::UniversalFallback, attr::MOI.ObjectiveSense,
+                 sense::MOI.OptimizationSense) where T
+    if sense == MOI.FEASIBILITY_SENSE
+        uf.objective = nothing
+    end
+    MOI.set(uf.model, attr, sense)
+end
+function MOI.get(uf::UniversalFallback,
+                 attr::MOI.ObjectiveFunctionType)
+    if uf.objective === nothing
+        return MOI.get(uf.model, attr)
+    else
+        return typeof(uf.objective)
+    end
+end
+function MOI.get(uf::UniversalFallback,
+                 attr::MOI.ObjectiveFunction{F})::F where F
+    if uf.objective === nothing
+        return MOI.get(uf.model, attr)
+    else
+        return uf.objective
+    end
+end
+MOI.supports(uf::UniversalFallback, ::MOI.ObjectiveFunction{F}) where F <: MOI.AbstractScalarFunction = true
```

Comment:
This is already covered by
https://github.com/JuliaOpt/MathOptInterface.jl/blob/74d74433c783a980fa118ffbda1fbe9f5f0f1d8c/src/Utilities/universalfallback.jl#L339